Open Judge System/Services/OJS.Services/Business/ParticipantScores/ParticipantScoresBusinessService.cs

```diff
@@ -376,13 +399,19 @@ private double CalculateTimeInMinutesFromParticipationStartToLastSubmission(Part
                 .Select(ps => new
                 {
                     ParticipantScore = ps,
-                    ProblemMaxPoints = ps.Problem.MaximumPoints
+                    ProblemMaxPoints = ps.Problem.MaximumPoints,
+                    Particinapnt = ps.Participant,
+                    totalScore = ps.Participant.Scores
+                        .Where(scores => !scores.Problem.IsDeleted)
+                        .Select(scores => scores.Points)
+                        .Sum()
```

Comment:
And this?

Reply:
Yes because there is a build error that says: Try specifying the type arguments explicitly.

Reply:
You cannot remove the totalScore from here?

Reply:
Done